src/loinclib/nlp_taxonomification.py

```diff
@@ -248,26 +262,39 @@ def _save_sssom(df: pd.DataFrame, outpath: t.Union[Path, str] = OUTPATH_MATCHES)
         },
     })
 
-    with open(outpath, 'w') as f:
+    # Save
+    with open(outpath_analysis, 'w') as f:
+        write_table(msdf, f)
+    with open(outpath_tree, 'w') as f:
```

Comment:
Changes to `nlp_taxonomification` to save file for easy access by `LoincTreeLoader`.

I'm saving 2 copies of the file. One in an 'analysis' dir along with some other files, but also one that can be accessed in the same dir as the other tree loader files.

@ShahimEssaid Perhaps this is not good design. I suppose I don't want a curator to edit the analysis file and then the tree file copy is what is read instead. Hmm...

But I felt like this was good for now. Because I might let you take it from here and create a different loader just for this NLP file.